### PR TITLE
1944 Input Tests

### PR DESCRIPTION
**Pull request by dhoanglong91epam**

### CODE REVIEW

**Comment by mf-code** [comment]
[5646] - aYzgBjiL

**Comment by mf-code** [comment]
[5646] - LVNlzOW9

**Comment by mf-code** [comment]
[5646] - Ts9MTpi9

**Comment by mf-code** [comment]
[5646] - IYZ70AVp

**Comment by mf-code** [comment]
[5646] - VGqknsVr

**Comment by mf-code** [comment]
[5646] - sZNCKrZh

**Comment by mf-code** [comment]
[5646] - 5ziKC0L0

**Comment by mf-code** [comment]
[5646] - 5dJbXEN5

**Comment by mf-code** [comment]
[5646] - VzzjoIeD

**Comment by mf-code** [comment]
[5646] - CJ1GeeTy

**Comment by mf-code** [comment]
[5646] - eaHal1Ec

**Comment by mf-code** [comment]
[5646] - pCTzz5ck

**Comment by mf-code** [comment]
[5646] - openjdk8 - Allure report on Netlify: https://5eeb417a3eaedd4b9309aa71--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5646] - openjdk10 - Allure report on Netlify: https://5eeb427624c20b2989a214a4--musing-swartz-6fc060.netlify.app

File: jdi-light-angular-tests/src/main/java/io/github/com/entities/Users.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package io.github.com.entities;
+
+import com.epam.jdi.tools.DataClass;
+
+public class Users extends DataClass<Users> {
+    public String firstName,
```
Comment (AkioFaris) [inline jdi-light-angular-tests/src/main/java/io/github/com/entities/Users.java:6]: Please change to 
    public String firstName;
    public String lastName;
    public String address;
    public String address2;
    public String city;
    public String state;
    public String postalCode;

Your variant it's not common for Java code https://rules.sonarsource.com/java/tag/convention/RSPEC-1659

File: jdi-light-angular-tests/src/main/java/io/github/com/entities/Users.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package io.github.com.entities;
+
+import com.epam.jdi.tools.DataClass;
+
+public class Users extends DataClass<Users> {
```
Comment (AkioFaris) [inline jdi-light-angular-tests/src/main/java/io/github/com/entities/Users.java:5]: Please rename Users to User. Entity class that describes a single entity should have singular value name.

File: jdi-light-angular-tests/src/test/java/io/github/epam/data/TestData.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package io.github.epam.data;
+
+
+import io.github.com.entities.Users;
+
+public class TestData {
+    public static Users DEFAULT_USER = new Users().set(c-> {
+        c.firstName = "Long"; c.lastName = "Dinh";
```
Comment (AkioFaris) [inline jdi-light-angular-tests/src/test/java/io/github/epam/data/TestData.java:8]: Maybe add a cosntructor to User class and use it here?

Comment (dhoanglong91epam) [reply jdi-light-angular-tests/src/test/java/io/github/epam/data/TestData.java:8]: I created Test Data using DataClass as per instruction in document. Please have a look: https://jdi-docs.github.io/jdi-light/?java#5-test-data

image

**Comment by mf-code** [comment]
[5659] - qdM6QIY8

**Comment by mf-code** [comment]
[5659] - 2fWlccn1

**Comment by mf-code** [comment]
[5659] - p1v1L4xv

**Comment by mf-code** [comment]
[5659] - x1J25rmL

**Comment by mf-code** [comment]
[5659] - QFIUwkWI

**Comment by mf-code** [comment]
[5659] - tgBo87jM

**Comment by mf-code** [comment]
[5659] - jPWsT3cF

**Comment by mf-code** [comment]
[5659] - IxpSfimu

**Comment by mf-code** [comment]
[5659] - zdPuAGH1

**Comment by mf-code** [comment]
[5659] - ijOy0QdM

**Comment by mf-code** [comment]
[5660] - KnJxGabH

**Comment by mf-code** [comment]
[5660] - hBeg32Pe

**Comment by mf-code** [comment]
[5660] - orJfuTbi

**Comment by mf-code** [comment]
[5659] - FqsZPlin

**Comment by mf-code** [comment]
[5660] - 3LoCTenv

**Comment by mf-code** [comment]
[5659] - dalmEOsG

**Comment by mf-code** [comment]
[5660] - iv9tj5Oy

**Comment by mf-code** [comment]
[5660] - boll19ZH

**Comment by mf-code** [comment]
[5659] - openjdk8 - Allure report on Netlify: https://5ef0352451dfb98e57b4da86--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5660] - 4usaGTFP

**Comment by mf-code** [comment]
[5660] - tbPIylV6

**Comment by mf-code** [comment]
[5659] - openjdk10 - Allure report on Netlify: https://5ef0361369e70d57c2a5042b--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5660] - GIGWZFsA

**Comment by mf-code** [comment]
[5660] - NBDy70b3

**Comment by mf-code** [comment]
[5660] - Y97youlQ

**Comment by mf-code** [comment]
[5660] - jRdvgXdz

**Comment by mf-code** [comment]
[5660] - openjdk8 - Allure report on Netlify: https://5ef03a67f3b4c8f8b00a9a3d--musing-swartz-6fc060.netlify.app

**Comment by mf-code** [comment]
[5660] - openjdk10 - Allure report on Netlify: https://5ef03b227588de740b84b4f5--musing-swartz-6fc060.netlify.app

**Pull request merged**